packages/server/graphql/mutations/upgradeToPro.ts

```diff
@@ -52,7 +52,9 @@ export default {
     try {
       await upgradeToPro(orgId, stripeToken, email)
     } catch (e) {
-      return standardError(e.param ? new Error(e.param) : e, {userId: viewerId, tags: e})
+      const param = (e as any)?.param
```

Comment:
let's put this in a conditional like
```ts
if ('param' in e) {
  const error = new Error(e.param)
  return ...
} else {
 ...
}
```

Reply:
Ah I tried to implement this but TypeScript complains that `Object is of type 'unknown'` on `if ('param' in e)`. If the two `any` casts in the catch are too ugly, an alternate option could be disabling an eslint rule that prevent us from doing `catch(e as any){`?